yarn.lock

```diff
@@ -18821,12 +18821,28 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/node@npm:^20.11.1":
-  version: 20.17.44
-  resolution: "@types/node@npm:20.17.44"
+"@types/node@npm:*, @types/node@npm:>=13.7.0, @types/node@npm:>=8.1.0, @types/node@npm:^22.13.5":
+  version: 22.15.19
+  resolution: "@types/node@npm:22.15.19"
   dependencies:
-    undici-types: ~6.19.2
-  checksum: 46ae4504ee490705ce9b19973ec9dc6581f3cbbd987bc64b939d7c24b1b401a55412de2376004790649b58551e098c6fb8077050c8acd4b5d9357678cfbca3b4
+    undici-types: ~6.21.0
+  checksum: 5aa45bf93c62bba1193bf07a56dddd84b74f89a554a08e1bd7772610f529d8dc6f828d464bc28c10001bd0092b6b3d11736544b52e154f25f5ca0ed93f04c32b
+  languageName: node
+  linkType: hard
+
+"@types/node@npm:^16.0.0":
```

Comment:
Any idea why we have this and ^18.0.0 below?

Reply:
yes @chenba  – everything 16 and 18 is due to Storybook's dependency of @types/node :
```
...
...
├─ @storybook/core-common@npm:7.4.5
│  └─ @types/node@npm:16.18.126 (via npm:^16.0.0)
│
├─ @storybook/core-common@npm:7.4.6
│  └─ @types/node@npm:16.18.126 (via npm:^16.0.0)
│
├─ @storybook/core-common@npm:7.5.3
│  └─ @types/node@npm:18.19.101 (via npm:^18.0.0)
...
...
```
Did I overhear @clouserw suggest we might remove storybook? 😉

Reply:
...three versions of `@storybook/core-common`? 🤨

Reply:
There's actually a lot more (12) 😅

Reply:
...our repo only imports it once though.  The rest are deps of other storybook components.